dockerfiles/start-cloudburst.sh

```diff
@@ -44,8 +44,9 @@ git checkout -b brnch origin/$ANNA_REPO_BRANCH
 git submodule sync
 git submodule update
 
+cd client/python && python3.6 setup.py install --prefix=$HOME/.local
```

Comment:
Hm, are we installing things in `$HOME/.local`, or are we installing them in `/usr/local`. I think `/usr/local` is probably better, but let’s make sure we are consistent.

Reply:
yeah I believe this is consistent with where it was installed previously in the docker image.

Reply:
👍